Dockerfile

```diff
@@ -7,5 +7,6 @@ RUN dnf update -y \
     --setopt=tsflags=nodocs \
     --setopt=deltarpm=false \
     python3-test \
+    python39 \
```

Comment:
This can be removed once https://github.com/fedora-python/fedora-python-tox/pull/29 is merged and released on dockerhub.

Reply:
the newest fedora-python-tox should contain Python 3.9 so you can remove this

Reply:
Done.